distribution/options.html

```diff
@@ -22,6 +22,7 @@
 		The token enables <a href="https://github.com/sindresorhus/refined-github/search?q=github-helpers+api" target="_blank">some features</a> to <strong>read</strong> data from public repositories
 	</p>
 	<ul>
+		<li data-validation data-scope="valid_token">The token is valid and it is 2 AM and I cannot think of better wording 💤 💤 💤
```

Comment:
```suggestion
		<li data-validation data-scope="valid_token">The token enables <a href="https://github.com/sindresorhus/refined-github/search?q=github-helpers+api" target="_blank">some features</a> to <strong>read</strong> data from public repositories

```

Reply:
One second I dont understand having a token with nothing enabled allows some features to run?

Reply:
Yes